src/main/java/org/breedinginsight/services/ProgramService.java

```diff
@@ -111,6 +128,7 @@ public Program create(ProgramRequest programRequest, AuthenticatedUser actingUse
                     .abbreviation(programRequest.getAbbreviation())
                     .objective(programRequest.getObjective())
                     .documentationUrl(programRequest.getDocumentationUrl())
+                    .brapiUrl(brapiUrl)
```

Comment:
Was talking with @DebWeigand and realized the default brapi URL should probably be saved to the database and not be left blank. If left blank, the .env default brapi URL could be changed and it would affect previously created programs.

Reply:
I think we could address this later.  Maybe make a card for this?

Reply:
Talked about this, going to leave and also change the brapiUrl to return "System Default" instead of the config default URL on programs without a custom brapi URL.

Reply:
Pushed changes